src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Welcome from 'components/header';
+
+const Index = () => <Welcome />;
```

Comment:
I prefer this (as used by Kehinde). Let our app have a root component called App

<img width="628" alt="Screenshot 2019-08-27 at 9 14 41 AM" src="https://user-images.githubusercontent.com/22734663/63753575-3a9f6c00-c8ab-11e9-8b74-1f8a01abbc4a.png">